test/integration/targets/apt/tasks/url-with-deps.yml

```diff
@@ -0,0 +1,43 @@
+- name: Install https transport for apt
+  apt:
+    name: apt-transport-https
+
+- name: Install deb file with dependencies from URL (check_mode)
```

Comment:
Make sure the package is not installed first, since the assert expects this task to report changed.

It's unlikely that the package will already be installed (especially with block/always for uninstall), but it's still a good idea to make sure that's the case.